nexus/completion.go

```diff
@@ -147,6 +152,9 @@ func (c *OperationCompletionSuccessful) applyToHTTPRequest(request *http.Request
 	if c.Header.Get(headerOperationStartTime) == "" && !c.StartTime.IsZero() {
 		request.Header.Set(headerOperationStartTime, c.StartTime.Format(http.TimeFormat))
 	}
+	if c.Header.Get(headerOperationCloseTime) == "" && !c.CloseTime.IsZero() {
+		request.Header.Set(headerOperationCloseTime, c.CloseTime.Format(time.RFC3339Nano))
```

Comment:
@cretz we wanted at least millisecond resolution here. Do you have a different time format in mind that would be easy to use across languages?
There isn't a standard time format with this resolution in the HTTP spec AFAICT.

Reply:
We plan to deprecate start time, which uses `http.TimeFormat`, and figure out a better approach to address the race for complete-before-start so I'm not worried about the inconsistency here.

Reply:
@cretz is this comment blocking the PR?
https://github.com/nexus-rpc/api/pull/24#pullrequestreview-3291042297